src/ResourceManager/RedisCache/Commands.RedisCache.Test/Commands.RedisCache.Test.csproj

```diff
@@ -213,5 +209,5 @@
   </ItemGroup>
   <ItemGroup />
   <Import Project="$(MSBuildToolsPath)\Microsoft.CSharp.targets" />
-  <Import Project="..\..\..\..\tools\Common.Dependencies.targets" />
+  <Import Project="..\..\..\..\tools\Common.Dependencies.Test.targets" />
```

Comment:
@maddieclayton I would check to make sure the assemblies in `Common.Dependencies.targets` are included in this `.csproj` file.